### DISCUSSION TITLE
Joint Use

### DISCUSSION THREAD

**Discussion by mochman** [karlomikus/bar-assistant#61] [Q&A] [answered]
Is this 1 user per "bar", or can multiple users be a part of the same "bar".  i.e. each user has a pool of the same ingredients and doesn't have to add them to their own shelf?

Or should this be posted over at [Salt Rim](https://github.com/karlomikus/vue-salt-rim)?

I am using the Docker All-in-One setup and every new user has to add the same ingredients to their shelf.

**Comment by zhdenny** [comment]
One Bar, Multiple Users will all be using the same database of ingredients and cocktails. Not sure if each user has their own inventory/shelf though. I think they do. You can set permissions to restrict users from making edits.

If you want multiple "bars" you can fire up multiple full instances of BarAssistant/SaltRim

**Comment by karlomikus** [comment, marked as answer]
All cocktails and ingredients are shared between users.
Every user has a personal shelf, shopping list and favorites.

**Reply by mochman** [reply]
Thanks, this is what I'm seeing in my installation.  People can see the same ingredients/cocktails, but have to add their own stuff to their separate shelf.  Guess I'll make a joint login so we can share the same shopping list.

**Comment by p-t-e-r** [comment]
It would be great to have an option to share the shelf/shopping list between users - but personal favorites.  Alternative is to make a shared login as mentioned above.
